### PR TITLE
Some updates for the breaking changes docs

### PR DESCRIPTION
**Pull request by Shazwazza**
(This is for review of a D-team member as part of our current sprint)

### CODE REVIEW

**Comment by marcemarc** [comment]
@Shazwazza In the Api Design part - if Apis do change - is it worth confirming here where the information regarding the change will be available? eg release notes and/or version specific upgrade guide?

File: Development-Guidelines/breaking-changes.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ _Defines what a breaking change is in regards to the Umbraco core codebase and d
 
 This section describes what a breaking change is in regards to the Umbraco codebase. Generally breaking changes are only made on major releases, however, in minor releases there may be changes to the codebase that some developers may consider 'breaking' as well. The following points describe what changes to the core codebase are or are not considered breaking changes.
 
+## Security
+
+If a security issue arrises with part of the Umbraco codebase that requires changing the behavior or signatures of the code and no other option is available to mitigate the security issue, then this type of breaking change will be released during a minor version of Umbraco. If this scenario occurs, there will be a documentation available for how to update your site to the security fixed version.
```
Comment (jmayntzhusen) [inline Development-Guidelines/breaking-changes.md]:
```suggestion
If a security issue arises with part of the Umbraco code base that requires changing the behavior or signatures of the code. If the only option is to mitigate the security issue, then this type of breaking change will be released during a minor version of Umbraco. If this scenario occurs, there will be documentation available for how to update your site to the secure fixed version.
```

Shortened sentences to improve readability.

File: Development-Guidelines/breaking-changes.md [hunk 2]
```diff
@@ -10,6 +10,18 @@ _Defines what a breaking change is in regards to the Umbraco core codebase and d
...
 
+## Security
+
+If a security issue arrises with part of the Umbraco codebase that requires changing the behavior or signatures of the code and no other option is available to mitigate the security issue, then this type of breaking change will be released during a minor version of Umbraco. If this scenario occurs, there will be a documentation available for how to update your site to the security fixed version.
+
+## Dependencies
+
+If a c# or JavaScript dependency is added or removed from the codebase, nuget package or exported release zip file, this is not considered a breaking change. We will mark this as "breaking" in the release notes to indicate the users that a dependency has changed. In the circumstance that a dependency is removed, it may be required that if you also require the removed dependency that you manually add it to your own project.
```
Comment (jmayntzhusen) [inline Development-Guidelines/breaking-changes.md]:
```suggestion
If a c# or JavaScript dependency is added or removed from the code base, a nuget package or the exported release zip file, it is not considered a breaking change. However, we will mark this as "breaking" in the release notes to indicate to the users that a dependency has changed. In the circumstance that a dependency is removed, you may have to manually add it to your own project if you depend on it.
```

**Comment by clausjensen** [comment]
@jmayntzhusen merging in your changes - apart from the shortened sentence. It still needs to make sense and the two if statements in two different sentences can be confusing - I'd rather have a long sentence then.

**Pull request merged**